src/components/LinkButton/LinkButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Pressable from '../Clickable/Clickable';
+
+const LinkButton = props => {
+    const openInNewTab = (url) => {
+        const newWindow = window.open(url, '_blank', 'noopener,noreferrer')
+        if (newWindow) newWindow.opener = null
+    }
+
+    return (
+        <Pressable
+            onClick={() => setTimeout(() => openInNewTab(props.url), 400)}
```

Comment:
I've never used this function, but I'm assuming it creates a 400 millisecond delay before executing `openInNewTab()`? If that's the case, it might be best to get rid of the delay as it can make opening the link feel a bit sluggish.